osu.Game.Rulesets.Osu/Difficulty/Skills/Aim.cs

```diff
@@ -14,46 +24,248 @@ namespace osu.Game.Rulesets.Osu.Difficulty.Skills
     /// </summary>
     public class Aim : Skill
     {
-        private const double angle_bonus_begin = Math.PI / 3;
-        private const double timing_threshold = 107;
+        private const double probability_threshold = 0.02;
```

Comment:
can you go through and add xmldoc for all constants that aren't immediately obvious?

Reply:
ok

Reply:
i've added xmldoc for constants and functions